bindings/java/src/org/sleuthkit/datamodel/CommunicationsUtils.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Sleuth Kit Data Model
+ *
+ * Copyright 2020 Basis Technology Corp.
+ * Contact: carrier <at> sleuthkit <dot> org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.sleuthkit.datamodel;
+
+import org.apache.commons.validator.routines.EmailValidator;
+
+/**
+ * Provides general utility methods related to communications artifacts.
+ * 
+ */
+public final class CommunicationsUtils {
```

Comment:
Tagging @bcarrier here since it's  change to public API and he needs to approve it.